app/src/main/java/com/randomappsinc/studentpicker/editing/RenameDialog.java

```diff
@@ -64,14 +65,22 @@ public RenameDialog(Context context, final Listener listener) {
                 .build();
     }
 
-    public void startRenamingProcess(String name, int maxAmount) {
+    private void handlePositiveAmountToRename() {
```

Comment:
This isn't worth the extra method. Especially in the neutral case where you already know exactly the amount to rename (it's just currentMaxAmount) and are incurring a performance loss from the integer parsing and string trimming.